modules/spout/renderablespherespout.cpp

```diff
@@ -31,10 +31,10 @@
 #include <openspace/util/sphere.h>
 #include <ghoul/opengl/texture.h>
 
-// The RenderableSphereSpout can be used to render a sphere with a texture that is
-// provided by another application on the same computer using the SPOUT library. Note that
-// this library is only available on Windows.
 namespace {
+    // The RenderableSphereSpout can be used to render a sphere with a texture that is
+    // provided by another application on the same computer using the SPOUT library.
+    // Note: The Spout library is only available on Windows
```

Comment:
```suggestion
    // Note: The SPOUT library is only available on Windows.
```

Reply:
The Spout people call it Spout:  https://spout.zeal.co/